tutorials/composite_mtbo.ipynb

```diff
@@ -0,0 +1,740 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Composite Bayesian Optimization with Multi-Task Gaussian Processes\n",
+    "\n",
+    "In this tutorial, we'll be describing how to perform multi-task Bayesian optimization over composite functions. In these types of problems, there are several related outputs, and an overall easy to evaluate objective function that we wish to maximize.\n",
+    "Multi-task Bayesian Optimization was first proposed by [Swersky et al, NeurIPS, '13](https://papers.neurips.cc/paper/2013/hash/f33ba15effa5c10e873bf3842afb46a6-Abstract.html) in the context of fast hyper-parameter tuning for neural network models; however, we demonstrate a more advanced use-case of [composite Bayesian optimization](https://proceedings.mlr.press/v97/astudillo19a.html) where the overall function that we wish to optimize is a cheap-to-evaluate (and known) function of the outputs. \n",
```

Comment:
Can you add a couple of sentences of motivation for why doing composite BO is valuable here? Using more information in the modeling yadda yadda